server/rooms.ts

```diff
@@ -118,6 +118,7 @@ export interface RoomSettings {
 	repeats?: RepeatedPhrase[];
 	autoModchat?: {rank: GroupSymbol, time: number, active: boolean};
 	tournaments?: TournamentRoomSettings;
+	detailsMod?: string;
```

Comment:
I would call this `defaultFormat` and plan for it to be eventually used for all relevant commands, such as `/learn`.